meta/Utils.m

```diff
@@ -169,6 +169,9 @@ occurrence of the given rule is replaced (if it exists) or added (if
 MathIndexToCPP::usage = "Converts integer-literal index from mathematica to c/c++ convention";
 FSPermutationSign::usage = "Returns the sign of a permutation given in a Cycles form";
 
+DecomposeVersionString::usage = "Return a list for string containing a version number";
+VersionOrder::usage = "";
```

Comment:
Can you please add a short description here? I think I'll not be able to remember which way around this comparison is (i.e. is it similar to < or >). :)

Alternatively you could also rename this function to `VersionLessThan` or so, in order to be more expressive, if you like.

Reply:
I've renamed `VersionOrder` to `VersionOrderGtEqThan`. I've also changed
```mathematica
If[!Utils`VersionOrder[sarahVersion, minimRequired], ....
```
to
```mathematica
If[!TrueQ@Utils`VersionOrderGtEqThan[sarahVersion, minimRequired], ....
```
because it had an annoying property that when someone removes the definition of `VersionOrderGtEqThan` from `Utils` then this block is never entered. Come to think of it, we should probably always use this `TrueQ` in all of our `If`s since we (almost?) always want them only to be either `True` or `False` and never this 3rd unevaluated one.

Reply:
Yes, this is good practice. In FS we've sometimes written
~~~.m
If[Func[expr] === True, ..., ...]
~~~
which is equivalent to your proposal, I think.